### PR TITLE
[Messenger] Fix DataCollector template

### PR DESCRIPTION
**Pull request by ottaviano**
| Q             | A
| ------------- | ---
| Branch?       | 4.2 
| Bug fix?      | yes
| New feature?  | no 
| BC breaks?    | no  
| Deprecations? | no 
| Tests pass?   | yes 
| Fixed tickets | ~
| License       | MIT
| Doc PR        | ~

It's for adding a closing DIV tag on L96 (+ re-indent the HTML code)

### CODE REVIEW

**Review by ro0NL — APPROVED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig [hunk 1]
```diff
@@ -71,31 +71,31 @@
...
-            <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount == messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
-
-            <div class="tab-content">
-                <p class="text-muted">Ordered list of dispatched messages across all your buses</p>
-                {{ helper.render_bus_messages(messages, true) }}
+        <div class="sf-tabs message-bus">
+            <div class="tab">
+                {% set messages = collector.messages %}
+                {% set exceptionsCount = collector.exceptionsCount %}
+                <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount == messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
```
Comment (Kronhyx) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig]: Taking advantage of the fact that you are refactoring why you do not use a strict comparison?
```suggestion
                <h3 class="tab-title">All<span class="badge {{ exceptionsCount ? exceptionsCount === messages|length ? 'status-error' : 'status-some-errors' }}">{{ messages|length }}</span></h3>
```

**Review by Kronhyx — CHANGES_REQUESTED** [review]
I have doubts

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig]: Twig doesn't have a `===` operator.

Comment (Kronhyx) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig]: haha ... ups ... I missed it ... hehe

**Review by Kronhyx — APPROVED** [review]

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig (hunk 1 above)
Comment (stloyd) [reply src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/messenger.html.twig]: But it has: 
```
exceptionsCount is same as (messages|length)
```
No sure it worth using it here anyway.

**Review by xabbuh — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @ottaviano.

**Pull request merged**